_new_content_templates/blank_releasenotes.md

```diff
@@ -25,11 +24,29 @@ These release notes provide an overview of the changes for {{site.productname}}
 
 {{site.releasenotes_for_stable}}
 
-## TinyMCE vnumtxt new features and enhancements
+## New features
 
-The following new features and enhancements were added for the {{site.productname}} vnumtxt release.
+The following new features were added for the {{site.productname}} vnumtxt release.
 
-### Improvement Name
+### Feature name
+
+### Minor new features
```

Comment:
This name is a little confusing, as it conflicts with semver "minor". I can't think of anything better though, so just leaving a comment in case anyone can think of something better.

Reply:
```suggestion
### Additional new features
```